config/helm/ec2-metadata-test-proxy/values.yaml

```diff
@@ -7,6 +7,7 @@ ec2MetadataTestProxy:
     scheduledEventStatus: active
     enableScheduledMaintenanceEvents: false
     enableSpotITN: true
+    enableIMDSV2: false
```

Comment:
why is this a value at all? it doesn't seem used in the code and even if it's set to false we seemingly try v2 in each request unless im missing something